server/api/app.py

```diff
@@ -59,7 +59,8 @@ def index():
         problem_url = pddl_files.url(filename_problem)
 
         # Test to call celery with a couple of solvers
-        solvers = {"lama","bfws"}
+
+        solvers = {"downward --alias lama-first"}
```

Comment:
Best stick with the upcoming `lama-first`. It'll work within 24hr most likely.

Reply:
Will replace that.